cmd/buildah/addcopy.go

```diff
@@ -247,6 +249,12 @@ func addAndCopyCmd(c *cobra.Command, args []string, verb string, iopts addCopyRe
 		InsecureSkipTLSVerify: systemContext.DockerInsecureSkipTLSVerify,
 		MaxRetries:            iopts.retry,
 	}
+	if iopts.parents {
+		options.ParentsPatterns = map[string]string{}
+		for _, pattern := range args {
+			options.ParentsPatterns[pattern] = pattern
```

Comment:
This doesn't appear to be doing anything different when `contextdir` is set, while https://github.com/containers/buildah/pull/6008/files#diff-f2e4566c6b7e38384283187aba6d7fd91e5ba8da2ffd0f849277bb76bff27fb3R561-R562 does.  If it's required, I'd expect it to be required in both places, and it's something that the integration tests should be verifying works correctly.